src/overlays/actors/ovl_Object_Kankyo/z_object_kankyo.c

```diff
@@ -32,41 +57,883 @@ const ActorInit Object_Kankyo_InitVars = {
     (ActorFunc)ObjectKankyo_Update,
     (ActorFunc)ObjectKankyo_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Object_Kankyo/ObjectKankyo_SetupAction.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Object_Kankyo/ObjectKankyo_Init.s")
+static u8 sIsSpawned = false;
+static s16 sTrailingFairies = 0;
+
```

Comment:
is there any indication in the game that the stuff floating around kokiri forest is supposed to be fairies?
The textures it uses is actually a chain of sun textures. Not that it should be named after that, but yeah lol. and Ive never heard anyone refer to them as fairies before. Thinking it may be adding room for confusion but idk
My preference would be something more generic like KokiriSparkleTrails or something along those lines

Reply:
idk if there's anything explicit, they just looked like fairies because of their behavior (waiting for the player to stand still to start circling, then scattering when the player moves)

Reply:
I think most people will read this and be thinking of actual fairies, like en_elf. So I think itd probably be best to go a little bit more generic. but we can take care of this in another documentation pass

Reply:
Can confirm, when I first read the code for reviewing I was thinking of fairy fountains fairies.
I'm not sure at which point I realized they actually were the particles in kokiri and lost woods.